examples/id_pool.rb

```diff
@@ -18,38 +18,38 @@
 ids = ['A2:32:C3:D0:00:00', 'A2:32:C3:D0:00:01']
 
 # Example: Disabling a ID Pool
-oneview_id_pool 'VMAC Pool' do
+oneview_id_pool 'VMAC Pool 1' do
   client my_client
   pool_type 'vmac'
   enabled false
   action :update
 end
 
 # Example: Enabling a ID Pool
-oneview_id_pool 'VMAC Pool' do
+oneview_id_pool 'VMAC Pool 2' do
   client my_client
   pool_type 'vmac'
   enabled true
   action :update
 end
 
 # Example: Allocating a list of the IDs
-oneview_id_pool 'VMAC Pool' do
+oneview_id_pool 'VMAC Pool 3' do
   client my_client
   pool_type 'vmac'
   id_list ids
 end
 
 # Example: Allocating a certain quantity of IDs
-oneview_id_pool 'VMAC Pool' do
+oneview_id_pool 'VMAC Pool 4' do
   client my_client
   pool_type 'vmac'
   count 2
   action :allocate_count
 end
 
 # Example: Removing a list of the IDs
-oneview_id_pool 'VMAC Pool' do
+oneview_id_pool 'VMAC Pool 5' do
```

Comment:
Why the every example is executed in a different ID Pool? Use only the necessary resources.